guerravc/KChallenge
# [FEAT] User can choose between List or Grid display

#### Purpose

#### Details
Now the user can choose how the movies are displayed between a list or a grid.

#### Steps to Test
* Run Application.
* Tap on the button in the top left corner.
* You would see now the grid view
